script.js

```diff
@@ -0,0 +1,47 @@
+// declare necessary elements
+
+const displayEl = document.querySelector('#calc-display');
+const numBtns = document.querySelectorAll('.number-button');
+const opBtns = document.querySelectorAll('.operation-button');
+const eqBtn = document.querySelector('.equals-button');
+const resetBtn = document.querySelector('#reset');
+
+// create a handler for the number buttons to add to the calculator display
+
+const text = document.createTextNode('')
+displayEl.appendChild(text);
+
+const handleNumAdd = (n) => {
+    const num = document.createTextNode(n.target.textContent);
+    return displayEl.appendChild(num);
+}; 
+
+// create a handler to run the equation on the display and display the
+
+const handleRunEquation = (n) => {
+    //save operator
+    const operator = 10
+    //save numbers and convert to number objects
+    const num1 = document.createTextNode(Number(displayEl.textContent.split('+' || '/')[0])); //split at operator

```

Comment:
this is starting to take shape like we discussed, consider saving the operator when the usert clicks on into a variable and then just split on that var, will make it much easier and dynamic. 

the other thing, dont worry about creating a textnode. you can just set innerText and it will parse into a tex tnode anyway. 

keep it up :)